### ISSUE TITLE
Export screen and depth buffer as an image

### ISSUE THREAD

**Issue by zoq** [dhewm/dhewm3#372]
Hello, is there a way to export the screen and depth buffer as an image e.g. using stb?

**Comment by nbohr1more** [comment]
Look at "RB_ShowDepthBuffer" in tr_rendertools.cpp

and

"FILL DEPTH BUFFER" in draw_common.cpp

**Comment by zoq** [comment]
@nbohr1more Thanks for the info; that's exactly what I was looking for. On another note, is there any way to get the motion vectors as well?

**Comment by nbohr1more** [comment]
It sounds like you may be interested in high quality motion blur or TxAA ? Doom 3 does not have a temporal buffer. 
For the player camera you could use neo/game/player.cpp idPlayer::CalculateFirstPersonView  You would have to go through game/anim and game/physics for other vector data

**Comment by zoq** [comment]
Correct, I was hoping to generate similar data I could get with something like https://bitbucket.org/Unity-Technologies/ml-imagesynthesis/src/master/.
